ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/ui/vr/RDXVRKinematicsStreamingMode.java

```diff
@@ -253,100 +282,120 @@ public void create(boolean createToolbox, KinematicsStreamingToolboxParameters k
+                                                                   controller.setAButtonText(streamToController.get() ? "Stop control" : "Start control");
+                                                                   if (streamToController.get())
+                                                                      controller.setBButtonText(showGhosts.get() ? "Hide ghosts" : "Show ghosts");
+                                                                   else
+                                                                      controller.setBButtonText("Relax hands");
+
+                                                                   InputDigitalActionData aButton = controller.getAButtonActionData();
+                                                                   InputDigitalActionData bButton = controller.getBButtonActionData();
+                                                                   InputDigitalActionData clickTriggerButton = controller.getClickTriggerActionData();
+                                                                   InputDigitalActionData leftJoystickButton = controller.getJoystickPressActionData();
+                                                                   boolean leftJoystickButtonClicked = leftJoystickButton.bChanged() && !leftJoystickButton.bState();
+
+                                                                   float joystickX = controller.getJoystickActionData().x();
+                                                                   float joystickY = controller.getJoystickActionData().y();
+
+                                                                   boolean leftAButtonPressed = aButton.bChanged() && !aButton.bState();
+                                                                   boolean leftBButtonPressed = bButton.bChanged() && !bButton.bState();
+                                                                   boolean leftTriggerPressed = clickTriggerButton.bChanged() && !clickTriggerButton.bState();
+                                                                   handleLeftControllerJoystickInput(leftAButtonPressed, leftBButtonPressed, leftTriggerPressed, leftJoystickButtonClicked, joystickX, joystickY);
+
+                                                                   // Check if left joystick is pressed in order to trigger recording or replay of motion
+                                                                   gripButtonsValue.put(RobotSide.LEFT, controller.getGripActionData().x());
+                                                                   kinematicsRecorder.recordControllerData(RobotSide.LEFT, leftAButtonPressed, leftBButtonPressed, leftTriggerPressed, joystickX, joystickY, controller.getAngularVelocity(), controller.getLinearVelocity(), getTrajectoryRecordFrame());
+                                                                   controllerLastPollTimeNanos = controller.getLastPollTimeNanos();
```

Comment:
It would be nice to left indent this block

Reply:
I've always had the opinion that the formatter should have wider lines, which you haven't been receptive to. The readability goes way down when there's excessive line line breaks, so I'll keep it as-is.

Reply:
Everywhere else in this file and the VR code, the controller blocks look like this. I was asking for the new code to follow that same style. I think maybe there was a miscommunication?
![image](https://github.com/user-attachments/assets/178ba525-ddad-4bde-a7ec-5a5ee4868718)

Reply:
Gotcha. The formatter has the brackets placed in that spot, so I'd prefer to at least partially stick with the formatter here as opposed to completely break from it

Reply:
Or to change the formatter to get the brackets there by default

Reply:
The formatter doesn't support that last time I checked, which is why it's like this, but I can check again